site/content/en/docs/manual/advanced/shortcuts.md

```diff
@@ -2,79 +2,57 @@
-| `Alt+K`                    | Performs zoom out                                                                                        |
+These shortcuts are organized by scopes. Some are global, meaning they work across the entire application, while others are specific to certain sections or workspaces. This setup allows for shortcut reuse in different scopes, depending on whether they might conflict. For example, global shortcuts must be unique since they apply across all pages and workspaces. However, similar shortcuts can be used in different workspaces, like having the same shortcuts in both the Standard Workspace and the Standard 3D Workspace, as these two do not coexist.
+
+| **Scope**                        | **Shortcut Conflicts**                                                                       |
+|----------------------------------|----------------------------------------------------------------------------------------------|
+| **Global**                       | Must be unique across all scopes, as they apply universally.                                 |
+| **Annotation Page**              | Must be unique across all scopes, except Labels Editor.                                      |
+| **Standard Workspace**           | Must be unique across itself, Annotation Page and Global Scope.                              |
+| **Standard 3D Workspace**        | Must be unique across itself, Annotation Page and Global Scope.                              |
+| **Attribute Annotation Workspace** | Must be unique across itself, Annotation Page and Global Scope.                            |
+| **Review Workspace**             | Must be unique across itself, Annotation Page and Global Scope.                              |
+| **Tag Annotation Workspace**     | Must be unique across itself, Annotation Page and Global Scope.                              |
+| **Control Sidebar**              | Must be unique across itself, all workspaces, Annotation Page and Global Scope.              |
+| **Objects Sidebar**              | Must be unique across itself, all workspaces, Annotation Page and Global Scope.              |
+| **Labels Editor**                | Must be unique across itself and Global Scope.                                               |
+
+## Shortcuts Customization
+
+You can customize shortcuts in the CVAT settings.
+
+- Open Settings:\
+![](/images/shortcuts01.png)
+
+- Go to the Shortcuts tab:\
+![](/images/shortcuts02.png)
+
+- You'll see the shortcuts customization menu:\
+![](/images/shortcuts03.png)
+
+- As it can be seen there is a warning, that some shortcuts are reserved by the browser and cannot be overridden by CVAT, there isn't a specific list available for such combinations, but shortcuts such as ctrl + tab (switching tabs) or ctrl + w (closing tabs) etc, are reserved by the browser and shortcuts such as alt + f4 (closing the window) are reserved by the operating system.
```

Comment:
```suggestion
- As it can be seen there is a warning, that some shortcuts are reserved by a browser and cannot be overridden in CVAT, there isn't a specific list available for such combinations, but shortcuts such as ctrl + tab (switching tabs) or ctrl + w (closing tabs) etc, are reserved by the browser and shortcuts such as alt + f4 (closing the window) are usually reserved by your operating system.
```